src/helpers.php

```diff
@@ -44,3 +44,9 @@ function str_contains(string $haystack, string | array $needles): bool
 
     return false;
 }
+
+function der2pem($der_data): string
```

Comment:
Let's not pollute the global namespace with this function, but instead add it to the `SslCertificate` class